lib/mysqlDao.js

```diff
@@ -42,11 +42,12 @@ class MysqlDao {
     get mysqlService() { return this._mysqlService; }
 
     /**
-     * @param {object} createArgs
-     * @returns {Promise.<int,Error>} - ID of last inserted item
+     * @param {object} createArgs column values to be inserted
+     * @param {boolean} [ignoreOnDuplicateKey] if true, insert will ignore duplicate keys
+     * @returns {Promise.<int,Error>} - ID of last inserted item; or 0 if ignoreOnDuplicateKey is true and nothing new was inserted
      */
-    create(createArgs) {
-        const sql = this._queryHelper.create(createArgs);
+    create(createArgs, ignoreOnDuplicateKey) {
```

Comment:
I would probably create an "options" object argument with ignoreOnDuplicateKey being one of the possible options.  Just in case we need to add more stuff in the future.